charts/gxf/templates/deployment.yaml

```diff
@@ -91,77 +90,42 @@ spec:
-
-        readinessProbe:
-          failureThreshold: 6
-          periodSeconds: 5
-          httpGet:
-            port: 9404
-            path: /metrics
-
-        livenessProbe:
-          failureThreshold: 6
-          periodSeconds: 5
-          httpGet:
-            port: 9404
-            path: /metrics
-{{- else }}
-{{- if .Values.startupProbe }}
-        startupProbe:
-{{ toYaml .Values.startupProbe | indent 10 }}
-{{- end }}
-{{- if .Values.readinessProbe }}
-        readinessProbe:
-{{ toYaml .Values.readinessProbe | indent 10 }}
-{{- end }}
-{{- if .Values.livenessProbe }}
-        livenessProbe:
-{{ toYaml .Values.livenessProbe | indent 10 }}
-{{- end }}
-{{- end }}
-{{- if .Values.httpdSidecar.enabled }}
+        {{- if .Values.extraVolumeMounts }}
+        {{- toYaml .Values.extraVolumeMounts | nindent 8 }}
+        {{- end }}
+        startupProbe: {{- toYaml .Values.startupProbe | nindent 10 }}
+        livenessProbe: {{- toYaml .Values.livenessProbe | nindent 10 }}
+        readinessProbe: {{- toYaml .Values.readinessProbe | nindent 10 }}
+      {{- if .Values.httpdSidecar.enabled }}
       - name: httpd
-        image: {{ .Values.httpdSidecar.imageRepo }}{{ .Values.httpdSidecar.image }}{{- if .Values.httpdSidecar.imageTag -}}:{{- end -}}{{ .Values.httpdSidecar.imageTag }}
-{{- if .Values.httpdSidecar.resources }}
-        resources:
-{{ toYaml .Values.httpdSidecar.resources | indent 10 }}
-{{- end }}
+        image: {{ .Values.httpdSidecar.imageRepo }}{{ .Values.httpdSidecar.image }}:{{ .Values.httpdSidecar.imageTag }}
+        {{- if .Values.httpdSidecar.resources }}
+        resources: {{- toYaml .Values.httpdSidecar.resources | nindent 10 }}
+        {{- end }}
         ports:
-        - containerPort: 443
+        - name: https
+          containerPort: {{ .Values.httpdSidecar.port }}
+          protocol: TCP
         volumeMounts:
         - name: apache-config
           mountPath: /usr/local/apache2/conf/httpd.conf
           subPath: httpd.conf
         - name: apache-config
-          mountPath: /usr/local/apache2/conf/extra/httpd-vhosts.conf
+          mountPath: /usr/local/apache2/conf/extra/vhosts.conf
           subPath: vhosts.conf
-        - name: apache-config
-          mountPath: /usr/local/apache2/conf/extra/httpd-ssl.conf
-          subPath: ssl.conf
         - name: apache-server-certs
           mountPath: /certs
-{{- if .Values.httpdSidecar.extraVolumeMounts }}
-{{ toYaml .Values.httpdSidecar.extraVolumeMounts | indent 8 }}
-{{- end }}
-{{- if .Values.httpdSidecar.readinessProbe }}
+        {{- if .Values.httpdSidecar.extraVolumeMounts }}
+        {{- toYaml .Values.httpdSidecar.extraVolumeMounts | nindent 8 }}
+        {{- end }}
         readinessProbe:
-{{ toYaml .Values.httpdSidecar.readinessProbe | indent 10 }}
-{{- end }}
-{{- if .Values.httpdSidecar.livenessProbe }}
-        livenessProbe:
-{{ toYaml .Values.httpdSidecar.livenessProbe | indent 10 }}
-{{- end }}
```

Comment:
Process 1 will indicate the liveness of the httpd process. This is checked by default in Kubernetes.
Restarting the pod if the pod doesn't respond also won't do much here since it is most likely a configuration error.